easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-5.1.1-2023b.yml

```diff
@@ -8,3 +8,8 @@ easyconfigs:
       options:
         # see https://github.com/easybuilders/easybuild-easyconfigs/pull/22371
         from-commit: 29f82e43229663c22f0c76cb3fc7b6dd5c407cd7
+  - RAxML-8.2.13-gompi-2023b.eb:
+      options:
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/23975
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/24627
+        from-commit: b1f428cb3d93eb8be2f6dce1494b23713daa3ecd
```

Comment:
@PetrKralCZ This PR has become stale, can you update this branch and move the change to the correct easystack file: `easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-5.1.2-2023b.yml`